### DISCUSSION TITLE
1440p Twitch stream showing 720p as best quality

### DISCUSSION THREAD

**Discussion by cmax110** [streamlink/streamlink#6310] [Q&A] [answered]
### Description

When trying to play a 1440p twitch stream through streamlink, I only get up to 720p for the best quality.

![image](https://github.com/user-attachments/assets/c73c35ed-1754-49b7-82fc-b5fe56d650ab)

I thought it might be the same issue as this post, https://github.com/streamlink/streamlink/issues/3158
Where it was a 1440p youtube stream, but since it was a DASH stream, it wasn't supported.

However looking at the video stats on twitch in a browser, it says HLS protocol not DASH. 
I did see that the 1440p source is H.265/HEVC and the 720p and below are H.264, so is this the reason?
And is there anyway I can get 1440p streams working through streamlink?

![image](https://github.com/user-attachments/assets/839f8133-4003-499a-a87d-d27959e5f3ed)

**Comment by bastimeyer** [comment, marked as answer]
https://github.com/streamlink/streamlink/issues/6145

**Reply by cmax110** [reply]
Thank you for this, I missed it in my searches, and looks like it isn't in the documentation. I am having an issue running that argument however, On the latest version of streamlink, tried uninstalling and reinstalling, and deleting my config file in my appdata. Any idea's what I am doing wrong?

![image](https://github.com/user-attachments/assets/89d3f00c-a364-4d04-b8e6-f8bb69e084bb)

**Reply by mkbloke** [reply]
#5769 has not been merged into master.

**Reply by cmax110** [reply]
Ah okay, my apologies, I have very little experience with github. I will wait for the new version then. Thank you as well for the help
